_posts/2020-11-11-hacktoberfest-sum-up.md

```diff
@@ -0,0 +1,39 @@
+---
+
+title: "Hacktoberfest 2020 sum-up"
+author: [molnarm84, huszerlbence]
+image: "../img/posts/hacktoberfest2020_sum-up.png"
+tags: [hacktoberfest, hacktoberfest2020, community, github, open source, JS, JavaScript, Cypress, end-to-end tests]
+redirect_to: https://www.sensenet.com/blog/2020-11-11-hacktoberfest-sum-up
+
+---
+
+Hacktoberfest 2020 is over, thanks for all the contributors. Check this year's stats and results.
+
+---
+
+sensenet joined #hacktoberfest celebration for the second time. This year, learned from our experiences back in 2019, we prepared a lot and created a dedicated site and much more to help contributors around the world🌍.
+
+This was a special year for us since we launched 🚀, along with hacktoberfest, our new software as a service model (called SNaaS).
```

Comment:
```suggestion
This was a special year for us since we launched 🚀, along with hacktoberfest, our new software as a service product (called SNaaS).
```